### ISSUE TITLE
Giseki Mobdrop - Erstpearl Crystals

### ISSUE THREAD

**Issue by CptA4** [MineInAbyss/Issue-Tracker#134]
### Related mob

Giseki

### Mobdrop name

Erstpearl Crystals

### Mobdrop Item

Amethyst Crystals or Emeralds

### Mobdrop Texture

The shape would resemble something similar to Bismuth Crystals as seen below.

![2f96628df52d532053ae0e0ae983ba8e](https://user-images.githubusercontent.com/48418904/168158255-82c153c9-b646-490a-8700-000e583b18e1.jpg)

The Color would resemble very closely to Opal as seen below.

![220px-10_7cts_Brazilian_Crystal_Opal](https://user-images.githubusercontent.com/48418904/168158305-e8cbc05a-6b3b-4514-999a-979313f78c2a.jpg)

### Additional info

They're called Erstpearls because Giseki primarily feed on Titanjaws, they eat the pearls Titanjaws have which harden into their shell and the Erst- prefix meaning formerly. This makes Erstpearl Crystals basically mean "Formerly Pearl Crystals". Could add this to the lore of this mobdrop, something like "Formed from Titanjaw Pearls" or something

### Agreements

- [X] I have searched for and ensured there isn't already an open suggestion similar to this.
- [X] I have ensured the mobdrop I'm requesting isn't already in the server.

**Comment by VaIienta** [comment]
Yeah seems fine, could probably be used as an ore for some special tools/items like amethysts are. Would need to finalise the mob first though.
